src/js/angular/repositories/controllers.js

```diff
@@ -772,9 +790,12 @@ function EditRepositoryCtrl($scope, $routeParams, toastr, $repositories, $locati
                     $scope.repositoryInfo.saveId = $scope.saveRepoId;
                     $scope.loader = false;
                 })
-                .error(function (data, status) {
+                .catch(function (response) {
+                    const data = response.data;
+                    const status = response.status;
                     if (status === 404 && $routeParams.repositoryId !== 'system') {
-                        toastr.error('Repo with name ' + '<b>' + $routeParams.repositoryId + '</b>' + ' doesn\'t exists', 'Error', {allowHtml: true});
+                        toastr.error('Repo with name ' + '<b>' + $routeParams.repositoryId + '</b>' + ' doesn\'t exists', 'Error',
```

Comment:
I know it's unrelated with the fix in this MR, but can you also i18n this label

Reply:
Added the label and also refactored the error handling a bit.